fabric/state.py

```diff
@@ -104,6 +104,13 @@ def _rc_path():
         help="abort instead of prompting (for password, host, etc)"
     ),
 
+    make_option('--banner-timeout',
+        type='int',
+        default=15,
```

Comment:
This should default to `None` so as not to break backwards compatibility - the current behavior is effectively "no timeout" since that's what Paramiko defaults to.